src/components/__snapshots__/KonnectorSuccess.spec.jsx.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`KonnectorSuccess should show banks if connector has datatypes with bankAccounts 1`] = `<Wrapper />`;
+
+exports[`KonnectorSuccess should show drive if trigger has a folder_to_save 1`] = `
+<Wrapper
+  folderId="deadbeef"
+/>
+`;
```

Comment:
Not sure if the snapshots are relevant in this state?

Reply:
Right :) Fixed, Using find().length instead